src/main/resources/templates/testRules.html

```diff
@@ -159,5 +157,26 @@ <h5 class="modal-title">Aggregated Object</h5>
 	    </div>
 	  </div>
 	</div>
+	<div id="infoModal" class="modal fade" role="dialog">
+	  <div class="modal-dialog" role="document">
+	    <div class="modal-content">
+	      <div class="modal-header">
+	        <h5 class="modal-title">Usage Information</h5>
+	        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+	          <span aria-hidden="true">&times;</span>
+	        </button>
+	      </div>
+	      <div class="modal-body modal-height-fit">
+	        <p id="infoContent" class="pre"></p>
+	        <div style="text-align: center">
```

Comment:
Boostrap has a class for this called text-center. Use that instead of style.

Reply:
Done